ext/random/tests/02_engine/pcgoneseq128xslrr64_jump_error.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+Random: Engine: PcgOneseq128XslRr64: Jump with negative $advance
+--FILE--
+<?php
+
+$engine = new \Random\Engine\PcgOneseq128XslRr64(1234);
+$referenceEngine = new \Random\Engine\PcgOneseq128XslRr64(1234);
+
+try {
+    $engine->jump(-1);
+} catch (ValueError $e) {
+    echo $e->getMessage(), PHP_EOL;
+}
+
+if ($engine->generate() !== $referenceEngine->generate()) {
+    die('failure: state changed');
+}
+
+die('success');
```

Comment:
Nit:
Please add a PHP closing tag. It makes it easier to run the test on its own.

Reply:
Done